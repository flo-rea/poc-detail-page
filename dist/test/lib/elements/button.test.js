"use strict";
const React = require('react');
const renderer = require('react/lib/ReactTestRenderer');
const button_1 = require('../../../src/lib/view/elements/button');
jest.mock('react-dom');
describe('Button', () => {
    it('has correct default options', () => {
        const component = renderer.create(React.createElement(button_1.default, {id: "button"}, "click-me."));
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=button.test.js.map