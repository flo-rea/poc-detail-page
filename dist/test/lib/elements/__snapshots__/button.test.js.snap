exports[`Button has correct default options 1`] = `
<span>
  <button
    id="button"
    type="button">
    click-me.
  </button>
</span>
`;
